### ISSUE TITLE
Problem with intellij

### ISSUE THREAD

**Issue by HexaNess** [lars-berger/GlazeWM#127]
Hi,
First of all I would like to congratulate you for this superb project, after 15min we know perfectly how to play with it, it's a real happiness.
However I have a problem that bothers me a little :(
With intellij I use a little known plugin called Glasscode (apparently it works with the other software of the jetbrain suite), to make it simple, it allows to have a transparency while coding, it's only aesthetic (but really beautiful).
So basically : 
- Intellij without this plugin : everything works fine
- Intellij with this plugin : the software crashes strangely as soon as you switch desktop

I would really like to use GlazeWM with the intellij plugin, can you take the time to test and see if something can be done? 
Thanks for reading, this was my first comment on github :)

**Comment by ParasiteDelta** [comment]
Could you expand a little bit and provide some more detail? Not a dev for GWM, but I went to try the listed configuration and it's working so far for me. How exactly does the program crash? Is IntelliJ crashing, or GWM? Do you mean switching Workspaces in GWM causes the crash, or switching virtual desktops in Windows? Are you on Win10 or Win11?

Test settings:
Win10
Intellij IDEA Community 2022.2
GlassCode
Fresh program install
GlazeWM 1.7.1
Tried switching Workspaces on panel with Intellij IDEA.

**Comment by HexaNess** [comment]
Hi, thanks a lot for answering.
However, I am very surprised that it works for you
I will try to explain the problem more precisely: 

When I run intellij (which has the glasscode plugin and with glazewm running) on a virtual desktop (created by glazewm) there is no problem I can code quietly, until I change desktop. When I change desktop then I come back to the desktop where I had intellij it is not possible to interact with it (intellij) I have to close it via the task manager to close it. So it's like the software crashed... So every time I want to code on intellij I have to turn off glazewm (or not activate the glasscode plugin) and that annoys me :(

Here is the configuration of my glasscode plugin : https://imgur.com/a/g9w0cIT
Thanks for the help :)

**Comment by ParasiteDelta** [comment]
Ah, very nice. Okay, so there are issues with Intellij resizing itself, and I've managed to mimic the issues. It's very peculiar. What seems to be happening is that the program doesn't quite handle the auto-hide (or whatever GWM does when switching workspaces) correctly, so you have to minimize the window and reopen it from the taskbar to get it working again. No idea why this happens, but it's part of why I hate using the JetBeans suite anyways, since they tend to be bloated and buggy, at least in my experience.

So, what can we do? Well, the only thing I can think of is to set the window to floating either via the config inside of your glaze-wm folder (<user>\Documents\.glaze-wm\config.yaml) as a rule for IDEA, or by using the shortcut Alt + Shift + Space, but this leads to sizing jank with IDEA, however it does remain functional after switching workspaces. Looking at the program itself, it appears to be using some kind of subprocess as a renderer, so there's not much that could be done.

Either way, try minimizing and restoring the window after swapping back, or just toggle floating and see what happens.
